content/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token.md

```diff
@@ -19,6 +19,11 @@ topics:
   - Access management
 shortTitle: Create a PAT
 ---
+{% note %}
+
+**Note:** [Git Credential Manager](https://github.com/GitCredentialManager/git-credential-manager/blob/main/README.md) is a secure, cross-platform alternative to using personal access tokens (PATs), which eliminates the need to manage PAT scope and expiration. For installation instructions, see [Download and install](https://github.com/GitCredentialManager/git-credential-manager/blob/main/README.md#download-and-install) in the GitCredentialManager/git-credential-manager repository.
```

Comment:
So sorry for the back and forth, but I didn't realize there's already a note right below this new one! Can you please combine then into one, like this?

**Notes**:

   - If you use {% data variables.product.prodname_cli %} to authenticate to {% data variables.product.product_name %} on the command line, you can skip generating a personal access token and authenticate via the web browser instead. For more information about authenticating with {% data variables.product.prodname_cli %}, see [`gh auth login`](https://cli.github.com/manual/gh_auth_login).
   - [Git Credential Manager](https://github.com/GitCredentialManager/git-credential-manager/blob/main/README.md) is a secure, cross-platform alternative to using personal access tokens (PATs), which eliminates the need to manage PAT scope and expiration. For installation instructions, see [Download and install](https://github.com/GitCredentialManager/git-credential-manager/blob/main/README.md#download-and-install) in the GitCredentialManager/git-credential-manager repository.

Thanks!

Reply:
No problem. I think I've got it fixed up. Just let me know if there's anything else you notice!